pkg/xmtp/listener.go

```diff
@@ -84,12 +85,12 @@ func (l *Listener) startMessageListener() {
 				msg, err := stream.Recv()
 				if err == io.EOF {
 					l.logger.Info("stream closed")
-					break
+					break streamLoop
```

Comment:
This is weird, which loop was it breaking out of? Couldn't be the outer one, was it?

Reply:
No, it was breaking out of the select